pow-migration/src/genesis/mod.rs

```diff
@@ -110,6 +109,21 @@ pub async fn get_pos_genesis(
             .await?
         };
 
+    // Calculate how much stake was burnt into registering validators and stakers
+    let mut burnt_registration_balance = genesis_stakers.iter().map(|staker| staker.balance).sum();
+    burnt_registration_balance += genesis_validators
+        .iter()
+        .map(|validator| validator.balance)
```

Comment:
Using the `fold` combinator here is the equivalent of `map().sum()` without mapping the struct first.